core/src/main/java/com/klaytn/caver/transaction/AbstractTransaction.java

```diff
@@ -250,32 +258,34 @@ public AbstractTransaction sign(String keyString, Function<AbstractTransaction,
     /**
      * Signs using all private keys used in the role defined in the Keyring instance.
      * It sets index and Hasher default value.
-     *   - signer : TransactionHasher.getHashForSignature()
+     * - signer : TransactionHasher.getHashForSignature()
+     *
      * @param keyring The Keyring instance.
      * @return AbstractTransaction
      * @throws IOException
      */
-    public AbstractTransaction sign(AbstractKeyring keyring) throws IOException  {
+    public AbstractTransaction sign(AbstractKeyring keyring) throws IOException {
         return this.sign(keyring, TransactionHasher::getHashForSignature);
     }
 
     /**
      * Signs using all private keys used in the role defined in the Keyring instance.
+     *
      * @param keyring The Keyring instance.
-     * @param signer The function to get hash of transaction.
+     * @param signer  The function to get hash of transaction.
      * @return AbstractTransaction
      * @throws IOException
      */
-    public AbstractTransaction sign(AbstractKeyring keyring, Function<AbstractTransaction, String> signer) throws IOException  {
-        if(this.getType().equals(TransactionType.TxTypeLegacyTransaction.toString()) && keyring.isDecoupled()) {
+    public AbstractTransaction sign(AbstractKeyring keyring, Function<AbstractTransaction, String> signer) throws IOException {
+        if (this.getType().equals(TransactionType.TxTypeLegacyTransaction.toString()) && keyring.isDecoupled()) {
             throw new IllegalArgumentException("A legacy transaction cannot be signed with a decoupled keyring.");
         }
 
-        if(this.from.equals("0x") || this.from.equals(Utils.DEFAULT_ZERO_ADDRESS)){
+        if (this.from.equals("0x") || this.from.equals(Utils.DEFAULT_ZERO_ADDRESS)) {
```

Comment:
Please fit to the current source code format.

Reply:
@sirano11 
Hi, I changed code format :)
Thanks for your feedback.